.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,52 @@
+name: Bug Report
```

Comment:
I'd recommend adding a note to have the latest version of LitGPT and up-to-date dependencies before submitting the issue.
Ideally to create a clean venv and install everything there with the command from README.

Reply:
Makes sense.
Let's start with something simple.

Reply:
Thanks for the feedback though!